src/mpi/mpi_caf.c

```diff
@@ -4876,6 +4876,14 @@ void PREFIX (change_team) (caf_team_t *team, int coselector __attribute__ ((unus
   MPI_Comm_size(*tmp_comm,&caf_num_images);
 }
 
+int
+PREFIX (team_number) (caf_team_t *team)
+{
+  if(used_teams->prev == NULL)
+    return -1;
+  return teams_list->team_id ;
+}
```

Comment:
Judging by the failures for GCC 6 on Travis-CI, I'm guessing you need to wrap this or something here with an `#ifdef GCC_7` or similar